### ISSUE TITLE
[5X_STABLE] "unrecognized format function type" warning in zic.c

### ISSUE THREAD

**Issue by d** [greenplum-db/gpdb#5939]
### Greenplum version or build
5X_STABLE (5.11.1 as of writing, commit 1f8b05f1dd)

### autoconf options used ( config.status --config )
CC=GCC 7.3.0

Get the following warning:
```
zic.c:129:1: warning: 'PG_PRINTF_ATTRIBUTE' is an unrecognized format function type [-Wformat=]
 static void verror(const char *string, va_list args) pg_attribute_printf(1, 0);
 ^~~~~~
zic.c:130:1: warning: 'PG_PRINTF_ATTRIBUTE' is an unrecognized format function type [-Wformat=]
 static void error(const char *string,...) pg_attribute_printf(1, 2);
 ^~~~~~
zic.c:131:1: warning: 'PG_PRINTF_ATTRIBUTE' is an unrecognized format function type [-Wformat=]
 static void warning(const char *string,...) pg_attribute_printf(1, 2);
 ^~~~~~

```

**Comment by gfphoenix78** [comment]
The missing macro seems to be fixed. Can we close this issue, Jesse?

**Comment by adam8157** [comment]
It's already fixed, ref: #5128. Feel free to reopen if it's not.
